jwtauthhandler.go

```diff
@@ -18,52 +21,118 @@ type JWTAuthHandler struct {
 	middleware *jwtmiddleware.JWTMiddleware
 }
 
+const signatureAlgorithm = validator.RS256
+
+var (
+	invalidSingingAlg         = fmt.Sprintf("invalid signing algorithm, expected %q", signatureAlgorithm)
+	ErrJWTInvalidCharacter    = errors.New("invalid character")
+	ErrJWTIssuerNotValid      = errors.New("iss not valid")
+	ErrJWTInvalidExpiry       = errors.New("token expired")
+	ErrJWTInvalidSigningAlg   = errors.New(invalidSingingAlg)
+	ErrJWTClaimFormatNotValid = errors.New("could not find claims")
+	ErrJWTInvalidToken        = errors.New("Authorization header format must be Bearer {token}")
+)
+
+// More finegrained ErrorHandler than the default.
+func ErrorHandler(w http.ResponseWriter, r *http.Request, err error) {
+	w.Header().Set("Content-Type", "application/json")
+
+	w.WriteHeader(http.StatusUnauthorized)
+
+	switch {
+	case errors.Is(err, ErrJWTInvalidCharacter):
+		_, _ = w.Write([]byte("invalid character"))
```

Comment:
waarom moet die string herhaald worden ipv `err.Error()` of `ErrJWTInvalidCharacter.Error()`

Reply:
Dat is omdat ik de error responses niet wilde wijzigen tov de vorige keer (en deze errormessages uit een andere go library komen die minder finegrained waren).